test/functional/commands/find/by-uiautomator-e2e-specs.js

```diff
@@ -11,111 +12,119 @@ describe('Find - uiautomator', function () {
+      .elementsByAndroidUIAutomator('new UiSelector().className("android.widget.TextView")');
     els.length.should.be.above(8);
     els.length.should.be.below(14);
   });
   it('should find elements without prepending "new UiSelector()"', async () => {
-    await driver.findElOrEls('-android uiautomator', '.clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('.clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should find elements without prepending "new UiSelector()"', async () => {
-    await driver.findElOrEls('-android uiautomator', '.clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('.clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should find elements without prepending "new UiSelector()"', async () => {
-    await driver.findElOrEls('-android uiautomator', 'clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should find elements without prepending "new "', async () => {
-    await driver.findElOrEls('-android uiautomator', 'UiSelector().clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('UiSelector().clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should ignore trailing semicolons', async () => {
-    await driver.findElOrEls('-android uiautomator', 'new UiSelector().clickable(true);', true)
+    await driver.elementsByAndroidUIAutomator('new UiSelector().clickable(true);')
       .should.eventually.have.length.at.least(10);
   });
-  it.skip('should find an element with an int argument', async () => {
-    let el = await driver.findElOrEls('-android uiautomator', 'new UiSelector().index(0)', false);
-    await driver.getName(el.ELEMENT).should.eventually.equal('android.widget.FrameLayout');
+  it('should find an element with an int argument', async () => {
+    let el = await driver.elementByAndroidUIAutomator('new UiSelector().index(0)');
+    await el.getAttribute('className').should.eventually.equal('android.widget.FrameLayout');
   });
   it('should find an element with a string argument', async () => {
     await driver
-      .findElOrEls('-android uiautomator', 'new UiSelector().description("Animation")', false)
+      .elementByAndroidUIAutomator('new UiSelector().description("Animation")')
       .should.eventually.exist;
   });
   it('should find an element with an overloaded method argument', async () => {
-    await driver.findElOrEls('-android uiautomator', 'new UiSelector().className("android.widget.TextView")', true)
+    await driver.elementsByAndroidUIAutomator('new UiSelector().className("android.widget.TextView")')
       .should.eventually.have.length.at.least(10);
   });
   it('should find an element with a Class<T> method argument', async () => {
```

Comment:
Do we really need identical tests with different descriptions?

Reply:
They're slightly different

```javascript

// This one uses classname as a string value
it('should find an element with an overloaded method argument', async () => {
    await driver.elementsByAndroidUIAutomator('new UiSelector().className("android.widget.TextView")')
      .should.eventually.have.length.at.least(10);
  });

  // This one uses a class instance (not a string)
  it('should find an element with a Class<T> method argument', async () => {
    await driver.elementsByAndroidUIAutomator('new UiSelector().className(android.widget.TextView)')
      .should.eventually.have.length.at.least(10);
  });
```

Reply:
Oh. Now I see.